persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/message/MessageDAOImpl.kt

```diff
@@ -413,6 +413,17 @@ internal class MessageDAOImpl internal constructor(
             .distinctUntilChanged()
     }
 
+    override suspend fun getConversationMessagesFromSearch(
+        searchQuery: String,
+        conversationId: QualifiedIDEntity
+    ): List<MessageEntity> = withContext(coroutineContext) {
+        queries.selectConversationMessagesFromSearch(
+            searchQuery,
+            conversationId,
+            mapper::toEntityMessageFromView
+        ).executeAsList()
+    }
```

Comment:
Cover this guy with tests? 

Just making sure that if we insert messages with "Hi Ananas", "Hello Bacon", "Ciao Cucumber", and search for "Bacon", only the second message is returned, etc.

Reply:
Oops! Missed this one!